evergreen/run-tests.sh

```diff
@@ -68,6 +70,16 @@ provision_compressor () {
 ############################################
 #            Main Program                  #
 ############################################
+echo "Initial MONGODB_CSFLE_PATH:" $MONGODB_CSFLE_PATH
+
+if [[ "$OS" =~ Windows|windows ]]; then
```

Comment:
this should not be here. You already initialize it in the `fetch-csfle-library.sh`

Reply:
Temp workaround.